vector-app/google-services.json

```diff
@@ -0,0 +1,378 @@
+{
```

Comment:
Ah no - the idea was to create a new project in firebase dedicated to this app, not use the existing SmarterDriver project in this.

We want to keep them split.